src/commands_old/check_node_status.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { createCommand } from '../utils/helpers';
+import getAPIClient from '../utils/api';
+
+const description = `Gets information about a node.
+
+	Example: check node status
+`;
```

Comment:
Just to discuss: I think it's easy to break spacing and tabs with this kind of usage. I'd prefer to write `\n` and `\t` specifically and avoid template literals if the text is not html or doesn't contain variables.

Reply:
Since this is displayed as it is on the UI, for me it make more sense to use the template.
However, in the `oclif` it's a bit different where we need to use the template